content/en/docs/examples/virtual-machines/multi-network/index.md

```diff
@@ -30,118 +30,122 @@ bare metal and the clusters.
-configure the Istio installation itself, and generate the configuration files
-that let VMs connect to the mesh. Prepare the cluster for the VM with the
-following commands on a machine with cluster admin privileges:
-
-1. Create a Kubernetes secret for your generated CA certificates using a command similar to the following. See [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/) for more details.
-
-    {{< warning >}}
-    The root and intermediate certificate from the samples directory are widely
-    distributed and known.  Do **NOT** use these certificates in production as
-    your clusters would then be open to security vulnerabilities and compromise.
-    {{< /warning >}}
-
-    {{< text bash >}}
-    $ kubectl create namespace istio-system
-    $ kubectl create secret generic cacerts -n istio-system \
-        --from-file=@samples/certs/ca-cert.pem@ \
-        --from-file=@samples/certs/ca-key.pem@ \
-        --from-file=@samples/certs/root-cert.pem@ \
-        --from-file=@samples/certs/cert-chain.pem@
+### Preparing your environment
+
+When expanding Istio's mesh capabilities to VMs across multiple networks (where the VM is in a network where traffic cannot directly route to pods in the Kubernetes cluster, for example), we'll need to take advantage of Istio's split-horizon DNS capabilities.
+
+Before we get started, you should prepare a VM and connect it to the Istio control plane through the Ingress Gateway. These steps are detailed in [Setup: Install: Virtual Machine Installation](/docs/setup/install/virtual-machine/).
+
+**Note** There are a few alterations to that document as follows:
+
+{{< warning >}}
+You must alter the VM set up instructions based on the suggestions in this section!
+{{< /warning >}}
+
+1. When we create the `IstioOperator` resource, we need to specify which networks we expect, and how to treat them
+1. When creating the `sidecar.env` we need to specify which network in which the VM belongs
+1. We need to create a `Gateway` resource to allow application traffic from the VM to the workload items running in the mesh
+
+#### Installing the Istio Control Plane
+
+When following the [Virtual Machine Installation](/docs/setup/install/virtual-machine/) setup guide to install the control plane, we will need to tweak the installation as follows:
+
+1. Specify the expected networks in the mesh, including the `vm-network`
+
+    {{< text bash yaml >}}
+$ cat <<EOF> ./vmintegration.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshExpansion:
+        enabled: true
+      multiCluster:
+        clusterName: kube-cluster
+      network: main-network
+      meshNetworks:
+        main-network:
+          endpoints:
+          - fromRegistry: kube-cluster
+          gateways:
+          - registryServiceName: istio-ingressgateway.istio-system.svc.cluster.local
+            port: 443
+        vm-network:
```

Comment:
Looks like this isn't passed through to the `ConfigMap`. Should be either `{}` or could be deleted, as far as I understand?

Reply:
thanks, will remove